exercises/option/option1.rs

```diff
@@ -18,6 +18,6 @@ fn main() {
             ((iter * 5) + 2) / (4 * 16)
         };
 
-        numbers[iter] = number_to_add;
+        numbers[iter as usize] = number_to_add;
```

Comment:
Happy to discuss this; here's the compiler message you get: 
```
21 |         numbers[iter] = number_to_add;
   |         ^^^^^^^^^^^^^ slice indices are of type `usize` or ranges of `usize`
   |
   = help: the trait `std::slice::SliceIndex<[std::option::Option<u16>]>` is not implemented for `u16`
   = note: required because of the requirements on the impl of `std::ops::Index<u16>` for `[std::option::Option<u16>]`
```

The message makes it sound like it _could_ have something to do with a SliceIndex of Option, but really this is about the type of the slice index. While this _is_ certainly a learning opportunity, it's not one that has anything to do with Option usage. 

I don't feel super strongly about it, but it was the place I got stuck the longest in the exercises!

Reply:
as the creator of the option1 exercise, I accept this change (looks like i forgot to put the as usize part myself)